satpy/writers/cf_writer.py

```diff
@@ -58,7 +58,7 @@ def omerc2cf(proj_dict):
 
 def geos2cf(proj_dict):
     """Return the cf grid mapping for the geos projection."""
-    grid_mapping_name = 'geostationary'
+    grid_mapping_name = 'vertical_perspective'
```

Comment:
why vertical perspective ? the correct projection is geos....

Reply:
I used the matching grid mapping name that are recognized in the cf grid mapping appendix (http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/apf.html)
And 'geos' gave me an error when I used the cfchecker....

Reply:
Strange, `geostationary` was introduced in 1.7